tensorflow/stream_executor/rocm/rocm_dnn.cc

```diff
@@ -43,6 +43,8 @@ limitations under the License.
 #include "rocm/include/miopen/miopen.h"
 // clang-format on
 
+#include "tensorflow/core/lib/hash/hash.h"
```

Comment:
nit: probably need to group it with other tensorflow headers

Reply:
will be fixed in next commit